### PR TITLE
Create reminder model

### PR DESCRIPTION
**Pull request by guillermosanta**

### CODE REVIEW

**Comment by guillermosanta** [comment]
In this branch, I created a new model called "Reminder" that includes the following fields: 
- channel: The id of the channel the reminder was created in
- userId: The id of the user that created the reminder (associated with the user model)
- body: The content of the reminder
- date: The date the reminder will fire

Reminders are a feature that is already implemented into the bot. Run `.help remind` to learn how the command works.

File: api/migrations/20210525230253-create-reminder.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"use strict";
```
Comment (chungthuang) [inline api/migrations/20210525230253-create-reminder.js:1]: The file name `20210525230253-create-reminder.js` should update to latest date.

File: api/migrations/20210525230253-create-reminder.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: {
+          model: "users",
+          key: "id",
+        },
+      },
+      body: {
+        allowNull: false,
+        type: Sequelize.TEXT,
```
Comment (chungthuang) [inline api/migrations/20210525230253-create-reminder.js:25]: Will there be limit on the body size?

File: api/migrations/20210525230253-create-reminder.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      date: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
```
Comment (chungthuang) [inline api/migrations/20210525230253-create-reminder.js:36]: Can reminders be deleted?

Comment (guillermosanta) [reply api/migrations/20210525230253-create-reminder.js:36]: Yes. Reminder will be deleted once the `date` is met, or if the user manually removes them using the `popReminder` command

File: api/migrations/20210525230253-create-reminder.js (hunk 2 above)
Comment (guillermosanta) [reply api/migrations/20210525230253-create-reminder.js:25]: Yes, but that is something that can be limited in the frontend.

File: api/migrations/20210525230253-create-reminder.js (hunk 1 above)
Comment (guillermosanta) [reply api/migrations/20210525230253-create-reminder.js:1]: I was confused in what repo I should run the `npx sequelize-cli model:generate` command, but I have figured it out now!

File: api/migrations/20210525230253-create-reminder.js (hunk 2 above)
Comment (chungthuang) [reply api/migrations/20210525230253-create-reminder.js:25]: By frontend you mean the bot logic?

**Review by chungthuang — APPROVED** [review]

**Pull request merged**